### ISSUE TITLE
DWS filter and profile issues

### ISSUE THREAD

**Issue by scd** [theCrag/website#3503]
From user ...

Hey Simon! I forgot my github account name so I though I'll just text you here. There seems to be a bug with DWS ascents or another issue:

This ascent is my only DWS send. I logged it in 2018. https://www.thecrag.com/ascent/1751457558

It does not appear on my profile. (Style DWS) Unlike on other profiles that logged this climb and also it does not seem that the number of maximum styles on profiles is limited. https://www.thecrag.com/climber/tobiau

Profile of someone else who also did this climb: https://www.thecrag.com/climber/caesar

Most interestingly it's also not shown in my "all ascents" lists when I filter for "style dws". https://www.thecrag.com/ascents/with-ascent-gear-style/dws/by/tobiau/?sortby=when-climbed,desc

**Comment by scd** [comment]
Initial triage

There are at least two bugs here. It seems as though the gear-style filter for dws is not working.

Secondly, the DWS tick type does not seem to work properly on profile. Relogging will not fix this. I have just tested in my account. I am not sure about the logic as to why it is not getting filtered in, but when I changed the tick type to onsight it got into my profile as a DWS.

 - [ ] why with-ascent-gear-style/dws is not working?
 - [ ] why DWS tick type not triggering profile inclusion in notable ascents.
